instrumentation/github.com/gocql/gocql/otelgocql/example/client.go

```diff
@@ -129,12 +129,8 @@ func main() {
 
 func views() ([]view.View, error) {
 	var vs []view.View
-	// TODO: Remove renames when the Prometheus exporter natively supports
-	// metric instrument name sanitation
-	// (https://github.com/open-telemetry/opentelemetry-go/issues/3183).
 	v, err := view.New(
 		view.MatchInstrumentName("db.cassandra.queries"),
-		view.WithRename("db_cassandra_queries"),
 	)
```

Comment:
These "blank action" views could be unified into a single view that matches `"*"`